README.md

```diff
@@ -4,6 +4,13 @@
 
 This project contains the application's UI built with React, Vite, TypeScript, and SWC. It requires Node.js v20, [nvm](https://github.com/nvm-sh/nvm) (or [n](https://github.com/tj/n)), and Yarn v4.
 
+## Repo Documentation
+
+- [Code of Conduct](./docs/CODE_OF_CONDUCT.md)
+- [Contributing](./docs/CONTRIBUTING.md)
+- [License](./docs/LICENSE.md)
+- [Security](./docs/SECURITY.md)
```

Comment:
Adding these links in the readme is a great idea. I'm suggesting an edit to keep them in the root folder:

```suggestion
- [Code of Conduct](./CODE_OF_CONDUCT.md)
- [Contributing](./CONTRIBUTING.md)
- [License](./LICENSE.md)
- [Security](./SECURITY.md)
```